Cargo.toml

```diff
@@ -58,7 +58,7 @@ hotshot = { git = "https://github.com/EspressoSystems/hotshot", tag = "0.5.74",
 # Hotshot imports
 hotshot-builder-api = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.74" }
 hotshot-builder-core = { git = "https://github.com/EspressoSystems/marketplace-builder-core", tag = "0.1.47" }
-marketplace-builder-core = { git = "https://github.com/EspressoSystems/marketplace-builder-core", tag = "0.1.47" }
+marketplace-builder-core = { git = "https://github.com/EspressoSystems/marketplace-builder-core", branch = "ab/update-module-path" }
```

Comment:
This should be updated before merge so we don't use a branch.

Reply:
done in 2f636f46f22e5c1a80d633326ec8456f11fb4281